Source/Core/Core/IOS/USB/Emulated/LogitechMic.h

```diff
@@ -0,0 +1,119 @@
+// Copyright 2025 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <atomic>
+#include <memory>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+#include "Core/IOS/USB/Common.h"
+#include "Core/IOS/USB/Emulated/Microphone-Logitech.h"
+
+namespace IOS::HLE::USB
+{
+enum LogitechMicrophoneRequestCodes
+{
+  REQUEST_SET_CUR = 0x01,
+  REQUEST_GET_CUR = 0x81,
+  REQUEST_SET_MIN = 0x02,
+  REQUEST_GET_MIN = 0x82,
+  REQUEST_SET_MAX = 0x03,
+  REQUEST_GET_MAX = 0x83,
+  REQUEST_SET_RES = 0x04,
+  REQUEST_GET_RES = 0x84,
+  REQUEST_SET_MEM = 0x05,
+  REQUEST_GET_MEM = 0x85,
+  REQUEST_GET_STAT = 0xff
+};
+
+enum LogitechMicrophoneFeatureUnitControlSelectors
+{
+  AUDIO_MUTE_CONTROL = 0x01,
+  AUDIO_VOLUME_CONTROL = 0x02,
+  AUDIO_BASS_CONTROL = 0x03,
+  AUDIO_MID_CONTROL = 0x04,
+  AUDIO_TREBLE_CONTROL = 0x05,
+  AUDIO_GRAPHIC_EQUALIZER_CONTROL = 0x06,
+  AUDIO_AUTOMATIC_GAIN_CONTROL = 0x07,
+  AUDIO_DELAY_CONTROL = 0x08,
+  AUDIO_BASS_BOOST_CONTROL = 0x09,
+  AUDIO_LOUDNESS_CONTROL = 0x0a
+};
+
+enum LogitechMicrophoneEndpointControlSelectors
+{
+  AUDIO_SAMPLING_FREQ_CONTROL = 0x01,
+  AUDIO_PITCH_CONTROL = 0x02
+};
```

Comment:
These three `enum`s are only needed in `LogitechMic.cpp`. Please move them there.
Also, please make them into `enum class` and use PascalCase naming for the members.